util/choose_postgresql_version

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -ex
+
+pg_short_version=pg"`pg_config --version | cut -d' ' -f2 | tr -d . | cut -c-2`"
+unlink src/lib.rs || true 
+(cd src; ln -s $pg_short_version.rs lib.rs)
```

Comment:
An alternative to this commit is an enhancement over what we discussed over the phone. If we had a wrapping rpg library, we could do this simply by setting the feature. See the autogenerated branch for an example.

You can build a pg94 version with
cargo build --features rpg/rpg94. I was mistaken before that this was a problem. The issue there is now we wouldn't get this cleaner calling convention for calling the raw library. In addition, we would have to go back to #cfg trickery if we wanted to call the native libraries (as opposed to the cleaner code in this commit)